src/app/static/src/app/components/figures/strategise/strategiesTable.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <b-table :items="items"></b-table>
+</template>
+
+<script lang="ts">
+import {BTable} from "bootstrap-vue";
+import Vue from "vue";
+import {StrategyWithThreshold} from "../../../models/project";
+
+const names: Record<string, string> = {
+  "irs": "IRS only",
+  "llin-pbo": "Pyrethroid-PBO ITN only",
+  "irs-llin-pbo": "Pyrethroid-PBO ITN with IRS*",
+  "llin": "Pyrethroid LLIN only",
+  "irs-llin": "Pyrethroid LLIN with IRS*",
+  "none": "No intervention"
+};
+
+const colours: Record<string, string> = {
```

Comment:
Nice use of the bootstrap classes, but it does look a bit eccentric! I'd vote for using a colour palette that matches those used in the charts a little more closely and creating our own css classes with those. Hm, I see you've created classes below, are 'primary' etc the only options the BTable will accept in cellVariants? Maybe a comment to this effect in that case. 

More generally, I guess colours and intervention labels are the sort of metadata values which is would make sense to fetch from mintr, since those values are already coming from mintr for the regions page. But that can be a future modification if desirable.

Reply:
Yep - colour variants are an expectation of BTable. Will add explanatory comment - thanks.

Reply:
7899934